src/components/Modals/SetAclPermissionsModal.jsx

```diff
@@ -0,0 +1,216 @@
+                <Typography
+                  variant="subtitle2"
+                  mb={2}
+                  sx={{
+                    width: '250px',
+                    whiteSpace: 'nowrap',
+                    overflow: 'hidden',
+                    textOverflow: 'ellipsis'
+                  }}
+                >
+                  <span style={{ color: 'grey' }}>File Name: </span>
+                  {dataset.documentName}
+                </Typography>
+              ) : null}
+              <FormControl required fullWidth sx={{ marginBottom: '1rem' }}>
+                <InputLabel id="permissionType-label">Select One</InputLabel>
+                <Select
+                  labelId="permissionType-label"
+                  id="permissionType"
+                  label="Select One"
+                  value={permissionState.permissionType}
+                  onChange={(e) =>
+                    setPermissionState({ ...permissionState, permissionType: e.target.value })
+                  }
+                  name="setAclPerms"
+                >
+                  <MenuItem value="Give">Give Permission</MenuItem>
+                  <MenuItem value="Revoke">Revoke Permission</MenuItem>
+                </Select>
+              </FormControl>
+              <br />
+              <FormControl fullWidth sx={{ marginBottom: '1rem' }}>
+                <TextField
+                  id="set-acl-to"
+                  name="setAclTo"
+                  value={permissionState.podUrlToSetPermissionsTo}
+                  onChange={(e) =>
+                    setPermissionState({
+                      ...permissionState,
+                      podUrlToSetPermissionsTo: e.target.value
+                    })
+                  }
+                  placeholder={permissionState.podUrlToSetPermissionsTo}
+                  label="Enter podURL"
+                  required
+                  error={permissionState.podUrlToSetPermissionsTo === podUrl}
+                  helperText={
+                    permissionState.podUrlToSetPermissionsTo === podUrl
+                      ? 'Cannot modify your permissions to your own pod.'.toUpperCase()
+                      : ''
+                  }
+                />
+              </FormControl>
+
+              {dataset.modalType === 'container' ? (
+                <DocumentSelection
+                  htmlForAndIdProp="set-acl-doctype"
+                  handleDocType={handleDocType}
+                  docType={docType}
+                />
+              ) : null}
+
+              <br />
+              <FormControl fullWidth sx={{ display: 'flex', gap: 2, flexDirection: 'column' }}>
+                <Button
+                  variant="contained"
+                  disabled={permissionState.podUrlToSetPermissionsTo === podUrl || state.processing}
+                  type="submit"
+                  color="primary"
+                  startIcon={<ShieldIcon />}
```

Comment:
While the shield is nice, I think the previous iteration with the gear looks better. It's certainly something I think about when it comes to a settings option or menu.

Reply:
I changed it because of what @andycwilliams said above. What should I do here?